index.js

```diff
@@ -1,300 +1,147 @@
-// Following the TFD spec (also known as TFK), the naming convention in this
+// Following the BFE TFD spec (also known as TFK), the naming convention in this
 // file uses "T" to mean "Type byte", "TF" to mean "Type byte and Format byte"
 // and "D" to mean "Data bytes".
 
-const { isFeedType, isMsgType, isBlobType } = require('ssb-ref')
-const TYPES = require('./bfe.json')
+const definitions = require('./bfe.json')
+const { decorateBFE, sortByNames, findClassicTypeFormat } = require('./util')
 
-function convertTypesToNamedTypes(TYPES) {
-  const NAMED_TYPES = {}
+const TYPES = decorateBFE(definitions)
 
-  function convertFormats(type) {
-    const formats = {}
-    for (let i = 0; i < type.formats.length; ++i) {
-      const format = type.formats[i]
-      formats[format.format] = format
-    }
-
-    return { ...type, formats }
-  }
-
-  for (let i = 0; i < TYPES.length; ++i) {
-    const type = TYPES[i]
-    NAMED_TYPES[type.type] = convertFormats(type)
-  }
-
-  return NAMED_TYPES
+const STRING = {
+  TF: Buffer.from([6, 0]),
```

Comment:
There's quite a lot of back and forth going on with refactors.

- [It used to be `Buffer.from([6, 0])`](https://github.com/ssb-ngi-pointer/ssb-bfe/blob/609c7e095c9f357b123552fdc8e8c4dca0dcc49b/index.js#L24)
- [We started using bfe.json to fetch the `code`s](https://github.com/ssb-ngi-pointer/ssb-bfe/blob/a097a14b1f5f914641d096fcb2ca9d943159ea77/index.js#L55)
- Now this PR is reverting to how it was in the beginning

I believe you could use `definitions` or `TYPES` here, right?

Further, I don't see a need to change `STRING_TF` to `STRING.TF`, is this just stylistic? Why not preserve the previous style?

Reply:
yeah it's totally styalistic. I think spacially some and like grouping things, and dots make much more sense to me that heaps of SNAKE_CASE_DESCRIPTIONS. I don't know what I find that slower to parse. I think for STRING it doesn't make that much sense, but I really like it for the BOOL case, and was trying to be consistent.

Reply:
I reverted it because I didn't understand the advantage of using magic strings over magic numbers. The problem with magic strings is if it was types wrong, you can _nothing_ out of the object.

I don't care that much about this change at the end of the day. Happy to revert if you prefer